### PR TITLE
Update docker installation options

### PR DESCRIPTION
**Pull request by buckaroogeek**
- Amends developer documentation to include a link to the Docker and Docker Composes installation guide in Fedora Quick Docs.
- Amends Compose description with an overview of important changes in Compose V2.

### CODE REVIEW

File: tools/docker/compose.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ order: 5
 
 # Docker Compose
 
-[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate Docker containers using a simple YAML file which describes your whole setup.
+[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate Docker one or more containers using a simple YAML file which describes your whole setup.
```
Comment (jackorp) [inline tools/docker/compose.md:9]: "orchestrate Docker one or more" does not seem to be in correct order

Would the following work here?
```suggestion
[Docker Compose](https://docs.docker.com/compose/) is a tool to orchestrate one or more Docker containers using a simple YAML file which describes your whole setup.
```

Not sure whether docker-compose works with podman. If it's sensible to say that it works in general with containers, then we could throw out the "Docker" word.

**Review by jackorp — CHANGES_REQUESTED** [review]
PTAL on the comment, otherwise content looks good, thanks!

File: tools/docker/compose.md (hunk 1 above)
Comment (buckaroogeek) [reply tools/docker/compose.md:9]: Thanks.  docker-compose will work with podman if podman-docker is installed.  I recommend 

`..to orchestrate one or more containers using a simple YAML file ...`

thank you

**Comment by jackorp** [comment]
LGTM, Thanks!

**Review by jackorp — APPROVED** [review]

**Pull request merged**